Signal/src/Models/TSMessageAdapaters/TSPhotoAdapter.m

```diff
@@ -29,7 +31,7 @@ @implementation TSPhotoAdapter
 
 - (instancetype)initWithAttachment:(TSAttachmentStream *)attachment incoming:(BOOL)incoming
 {
-    self = [super initWithImage:attachment.image];
+    self = [super initWithImage:nil];
```

Comment:
Same idea; don't load images until we need them and only retain them while they're in use.